UnityProject/Assets/Scripts/Sprite Handler/SpriteHandlerController.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using Mirror;
+
+/// <summary>
+/// The main controller for each component of its Sprites 
+/// </summary>
+public class SpriteHandlerController : NetworkBehaviour
+{
+
+	private Pickupable pickupable;
+	private ItemAttributesV2 itemAttributes;
+
+
+	/// <summary>
+	/// This needs to be changed To support multiple
+	/// </summary>
+	[Tooltip("Fill this out if you dont want it automatically picking the spriteHandler for you ")]
+	[SerializeField]
+	private SpriteHandler spriteHandler;
+
+	/// <summary>
+	/// Fall-back for old Objects
+	/// </summary>
+	private SpriteRenderer spriteRenderer;
+
+	private bool Initialised;
+
+	private void Awake()
+	{
		if (spriteHandler == null)
```

Comment:
Move the logic here in awake into its own method so that it can be called again correctly later on in your script